app/routes/goal_routes.py

```diff
@@ -0,0 +1,87 @@
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from app.routes.task_routes import validate_model
```

Comment:
It makes a little more sense to me to define validate_model in a separate helper file because the method is meant to be generic and can be applied to different models. validate_model isn't associated with tasks. If you define the method in a helper.py file, then you can import the method into task_routes.py and goal_routes.py